modules/serverless-kafka-sink-security-config.adoc

```diff
@@ -0,0 +1,91 @@
+// Module is included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sink-security-config_{context}"]
+= Configuring security for Kafka sinks
+
+_Transport Layer Security_ (TLS) is used by Apache Kafka clients and servers to encrypt traffic between Knative and Kafka, as well as for authentication. TLS is the only supported method of traffic encryption for Knative Kafka.
+
+_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resources (CRs) are installed on your {product-title} cluster.
+* Kafka sink is enabled in the `KnativeKafka` CR.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have a Kafka cluster CA certificate stored as a `.pem` file.
+* You have a Kafka cluster client certificate and a key stored as `.pem` files.
+* You have installed the OpenShift (`oc`) CLI.
+* You have chosen the SASL mechanism to use, for example, `PLAIN`, `SCRAM-SHA-256`, or `SCRAM-SHA-512`.
```

Comment:
Suggestion: remove "to use" and add semicolon ("You have chosen the SASL mechanism; for example, ...."

Reply:
I think this is OK as it is